.github/workflows/mobile-perf.yml

```
@@ -62,16 +62,8 @@ jobs:
             //test/performance:test_binary_size
           # Ensure files don't leak back into the main binary
           source: >-
-            rm
-            source/common/http/route_config_update_requster.h
-            source/common/listener_manager/listener_manager_impl.h
-            source/server/overload_manager_impl.cc
-            source/common/network/listen_socket_impl.h
-            source/common/network/tcp_listener_impl.h
-            source/server/guarddog_impl.h
-            source/server/watchdog_impl.h
-            source/server/options_impl.cc
-            source/extensions/access_loggers/common/file_access_log_impl.h
+            # TODO replace with tail -n + mobile/test/performance/files_em_does_not_use
+            rm `tail -n +2 mobile/test/performance/files_em_does_not_use`
```

```suggestion
            TOREMOVE=$(tail -n +2 mobile/test/performance/files_em_does_not_use)
            rm $TOREMOVE
```

current way would swallow any errors and backticks are strongly discouraged for exec in bash

even better would be to use something like the xargs way i posted offline (~relatedly) as that would even pass shellcheck